src/App.tsx

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
 
 import './App.scss';
+import { PostList } from './components/PostList';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+
+const preparedPosts = posts.map((post) => {
+  const userLink = users.find(user => post.userId === user.id);
+  const commentsArray = comments.filter(comment => post.id === comment.postId);
+
+  return {
+    user: userLink || null,
+    comments: commentsArray || null,
```

Comment:
It can't be null cause an empty array will be true anyway
```suggestion
    comments: commentsArray,
```